apps/docs/src/theme/Footer/Links/Simple/index.tsx

```
@@ -13,7 +13,7 @@ function SimpleLinkItem({ item }: { item: Props['links'][number] }) {
         <span
             className={clsx('footer__link-item', item.className)}
             // Developer provided the HTML, so assume it's safe.
-            // eslint-disable-next-line react/no-danger
+
             dangerouslySetInnerHTML={{ __html: item.html }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)